packages/node_modules/@webex/plugin-meetings/src/meeting/util.js

```diff
@@ -525,4 +525,21 @@ MeetingUtil.endMeetingForAll = (meeting) => {
     });
 };
 
+MeetingUtil.canEnableClosedCaption = (displayHints) => displayHints.includes(DISPLAY_HINTS.CAPTION_START);
+
+MeetingUtil.canStartTranscribing = (displayHints) => displayHints.includes(DISPLAY_HINTS.TRANSCRIPTION_CONTROL_START);
```

Comment:
@Coread  i still feel that we should not use the display hit and use the controls to determine if transcribing can be enabled or disabled

Reply:
Was this with regards to whether the display hint truly represents the availability of the transcribing?

Reply:
lets check this once with them , As developers it would be better to know what the meeting is capable of before joining